avocado/utils/archive.py

```diff
@@ -313,6 +418,18 @@ def close(self):
         self._engine.close()
 
 
+def is_bzip2_file(path):
+    """
+    Checks if file given by path has contents that suggests bzip2 file
+    """
+    try:
+        with open(path, "rb") as bz2_file:
+            # Check for bzip2 magic bytes (BZh)
+            return bz2_file.read(3) == b"BZh"
```

Comment:
In some places we have constants for the magic bytes, in others, functions that actually check for them.  It'd be nice to have a common approach.